deeppavlov/core/commands/infer.py

```diff
@@ -43,9 +42,7 @@ def build_model(config: Union[str, Path, dict], mode: str = 'infer',
     import_packages(config.get('metadata', {}).get('imports', []))
 
     model_config = config['chainer']
-
-    model = Chainer(model_config['in'], model_config['out'], model_config.get('in_y'))
-
+    model = Chainer(model_config['in'], model_config['out'], model_config.get('in_y'), model_config.get('buckets'))
```

Comment:
get buckets and histogram name from model_config['metadata']['profiling']
If there is no such field, don't initialize histogram, if there is such dict with parameters, initialize histogram here.
Initialized histogram should be passed to Chainer instead of buckets.

Reply:
so statistics will be valid in inference mode only

Reply:
initialize hist in case "profiling" exist without "buckets" ?